.github/dependabot.yml

```
@@ -35,3 +35,6 @@ updates:
         update-types:
           - "minor"
           - "patch"
+        exclude-patterns:
+          - "pandas"
+          - "scipy"
```

Thanks @kenibrewer ! I think we may want to include `sqlalchemy` as well based on what we're seeing in https://github.com/cytomining/pycytominer/pull/469 and the general change trajectories for Pandas + SQLAchemy (which aren't always aligned). The use of Pandas through Pycytominer is coupled to SQLAlchemy through SQLite data sourcing.
```suggestion
          - "scipy"
          - "sqlalchemy"
```